lessons/module-3/unit-testing-react.md

```diff
@@ -350,18 +356,19 @@ ternary or an `&&` condition in a template literal interpolation in `Grocery.js`
 <article className={`Grocery ${ starred && 'starred' }`}>
 ```
 
-Try both of those out and verify that they get the test passing. Then let the uneasy feeling 
-settle in as you consider that as time goes on, you'll have to do this repeatedly — first 
-with `purchased` and then possibly with more properties as requirements change down the line.
+Recognize that as your app develops, you will write a similar test for the 
+`purchased` className. Then again, for more properties as the requirements for the app extend to more features.
+
+### Add Classnames Package
 
-**Stop and Read**: To make our lives easier, we'll use the [classnames](https://www.npmjs.com/package/classnames) 
-package from npm. Check out the documentation before moving forward. You can install it with 
-`npm install -S classnames`.
+**Stop and Read**:  [classnames](https://www.npmjs.com/package/classnames) 
+To make our lives easier, use the classnames package from npm. Check out the documentation 
+before moving forward. You can install it with:
+```npm install -S classnames```
```

Comment:
Why so many ticks?

Reply:
Same idea: The triple ticks give the gray background that helps as visual placeholders as a student may be coding along.